inc/ThirdParty/Themes/Avada.php

```diff
@@ -0,0 +1,124 @@
+<?php
+namespace WP_Rocket\ThirdParty\Themes;
+
+use WP_Rocket\Admin\Options_Data;
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Compatibility class for Avada theme
+ */
+class Avada implements Subscriber_Interface {
+	/**
+	 * Options instance
+	 *
+	 * @var Options_Data
+	 */
+	private $options;
+
+	/**
+	 * Return an array of events that this subscriber wants to listen to.
+	 *
+	 * @since  3.3.1
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		$current_theme = wp_get_theme();
+
+		if ( 'Avada' !== $current_theme->get( 'Name' ) ) {
```

Comment:
```suggestion
		if ( 'Avada' === $current_theme->get( 'Name' ) || 'Avada' === $current_theme->get_template() ) {
```

I think we need to do that to work with child themes

Reply:
Hem. @Tabrisrp I think this is a valid point. What do you think? Should this case be handled too?

Reply:
yes that's true, and to be sure you should use `strtolower` around the returned values from the methods, + use `avada` with no caps, to cover all cases